tests/components/media_player/test_init.py

```diff
@@ -298,10 +298,20 @@ async def test_enqueue_alert_exclusive(hass: HomeAssistant) -> None:
         )
 
 
+@pytest.mark.parametrize(
+    "media_content_id",
+    [
+        "a/b c/d+e%2Fg{}",
+        "a/b c/d+e%2D",
+        "a/b c/d+e%2E",
+        "2012-06%20Pool%20party%20%2F%20BBQ",
+    ],
+)
```

Comment:
There was a problem here but its fixed by rc1 but I left the additional test cover